charter.md

```diff
@@ -22,14 +22,19 @@ Server-initiated services are out of scope. The working group will first deliver
 a requirements document for IP proxying. Once both are complete, the working group will focus on a protocol solution for
 IP proxying.
 
-The working group will consider fallback to versions of HTTPS that operate over TCP as a mitigation to UDP or HTTP/3
-blocking. Moreover, the working group will consider implications of tunneling protocols with congestion control and loss
-recovery over MASQUE, and may issue recommendations accordingly. New congestion control and loss recovery algorithms are
+Exercising the extension points defined by CONNECT-UDP and CONNECT-IP is important to ensure their continued useability.
```

Comment:
I'm not convinced that "useability" is a word.  Also, if the goal is to ensure that extensions are safe to negotiate (i.e. not prevented by server intolerance), that is quite different from "providing missing functionality" or "enhancing performance".

Reply:
Fair point. I left a suggestion that tries to clarify the intent of bringing extensions into the group.

Reply:
Good call, updated with Chris's text in [8af81ca](https://github.com/ietf-wg-masque/wg-materials/pull/7/commits/8af81ca9873dc5f769a87707af732f24be6455bf)